.github/workflows/e2e-tests.yml

```
@@ -44,171 +44,220 @@ jobs:
           MATRIX="$(node .github/files/e2e-tests/e2e-matrix.js)"
           echo "$MATRIX"
           echo "matrix=$MATRIX" >> "$GITHUB_OUTPUT"
-
-  e2e-tests:
-    name: "${{ matrix.project }} e2e tests"
+  build-projects:
```

Just a little formatting
```suggestion

  build-projects:
```